etc/ci.sh

```diff
@@ -37,18 +31,62 @@ else
         make
         cd ../..
 
-        sh bin/gap.sh --cover coverage tst/${TEST_SUITE}.g
+        sh bin/gap.sh -q tst/extractmanuals.g
 
-        # generate coverage report
-        sh bin/gap.sh -q <<GAPInput
-            if LoadPackage("profiling") <> true then
-                Print("ERROR: could not load profiling package");
-                FORCE_QUIT_GAP(1);
-            fi;
-            OutputJsonCoverage("coverage", "coverage.json");
+        for ch in tst/testmanuals/*.tst
+        do
+            COVNAME="coverage.`basename $ch .tst`"
+            sh bin/gap.sh -q --cover $COVNAME <<GAPInput
+            Read("tst/testmanuals.g");
+            TestManualChapter("$ch");
             QUIT_GAP(0);
 GAPInput
-    fi
-    cd bin/x86* ; gcov -o . ../../src/*
-    cd ../..
-fi;
+
+            sh bin/gap.sh -q <<GAPInput
```

Comment:
Oh wait, we call gap twice per iteration? Then it is 4 minutes :-).

Though perhaps we could move this second invocation out of the loop, turning it into a single invocation, where GAP loops over the `coverage.*.tst` files? This way, we turn ~80 GAP starts into a single one.